pom.xml

```diff
@@ -28,7 +28,7 @@
   <description>A Collection of Extensions for Syndesis</description>
 
   <properties>
-    <syndesis.version>2.0-SNAPSHOT</syndesis.version>
+    <syndesis.version>1.11.0</syndesis.version>
```

Comment:
We said this should be pointing to latest snapshot in master, right? I'd expect this to be `1.12-SNAPSHOT` and have `1.11.0` in `1.11.x` extension branch.

Reply:
Sure, once merged I am going to create 1.11.x branch and set 1.12-SNAPSOT to master.